bin/python/ripple/build/module/Default.py

```diff
@@ -0,0 +1,215 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals)
+
+import os
+
+from beast.build.Module import Module
+
+from beast.build.Function import Env, compose, for_tags, not_tags
+
+Append = Env.Append
+Prepend = Env.Prepend
+
+
+MODULE = Module(
+    files=compose(
+        for_tags(
+            'debug',
+            Append(CPPDEFINES=['DEBUG', '_DEBUG']),
+        ),
+
```

Comment:
What does "for_tags" mean, grammatically? Is that like a for_loop? What's a "for"? Whats "tags"?